### PR TITLE
RD-2789 Add page size configuration for Deployments View

### PR DESCRIPTION
**Pull request by Gelio**
This PR adds the page size configuration to the Deployments View widget and refactors the code a bit given the changes from https://github.com/cloudify-cosmo/cloudify-ui-components/pull/76.

## To do

- [ ] Restart the multibranch CI after https://github.com/cloudify-cosmo/docs.getcloudify.org/pull/1906 is merged to fix the `docs` check

## Video

https://user-images.githubusercontent.com/889383/127291300-34c6ba15-3932-4841-84e6-b15bb9738c47.mp4

## System tests

https://jenkins.cloudify.co/blue/organizations/jenkins/Stage-UI-System-Test/detail/Stage-UI-System-Test/997/pipeline

It uses the `6.1.0-build` Cloudify Manager branch, not `master`, but it should not matter.

Queued 2021-07-28 15:37 CEST

### CODE REVIEW

File: widgets/common/src/deploymentsView/configuration.ts [hunk 1]
```diff
@@ -39,7 +39,12 @@ export const sharedConfiguration: Stage.Types.WidgetConfigurationDefinition[] =
         default: deploymentsViewColumnIds.filter(columnId => columnId !== 'environmentType'),
         type: Stage.Basic.GenericField.MULTI_SELECT_LIST_TYPE
     },
-    Stage.GenericConfig.PAGE_SIZE_CONFIG(50),
+    {
+        ...Stage.GenericConfig.PAGE_SIZE_CONFIG(50),
+        hidden: false,
+        name: configurationT('pageSize.name'),
```
Comment (qooban) [inline widgets/common/src/deploymentsView/configuration.ts:45]: What do you think about converting the name to follow https://cloudifysource.atlassian.net/wiki/spaces/RD/pages/1875083446/UI+Translation+Files+Guidelines?

```suggestion
        name: tConfiguration('pageSize.name'),
```

Or would it be too big change not fitting this PR?

**Review by qooban — COMMENTED** [review]
Looks good! Great that it is possible to move forward with RD-1837.

Added just one comment. From my perspective nothing is blocking the approval when you bump cloudify-ui-components version.

File: widgets/common/src/deploymentsView/configuration.ts (hunk 1 above)
Comment (Gelio) [reply widgets/common/src/deploymentsView/configuration.ts:45]: Sure thing, I'll do it :slightly_smiling_face: Good catch

Comment (Gelio) [reply widgets/common/src/deploymentsView/configuration.ts]: Done :slightly_smiling_face:

**Review by stepek — COMMENTED** [review]
Looks good for me.

**Comment by Gelio** [comment]
I have updated the ui-components to the version that contains https://github.com/cloudify-cosmo/cloudify-ui-components/pull/76.

Thus, the PR is ready for final review.

Here's a run of the system tests for this branch: https://jenkins.cloudify.co/blue/organizations/jenkins/Stage-UI-System-Test/detail/Stage-UI-System-Test/997/pipeline

**Review by qooban — APPROVED** [review]
Looks good!

Thanks for applying small translation-related change.

**Comment by Gelio** [comment]
Hah, surprisingly, https://github.com/cloudify-cosmo/docs.getcloudify.org/pull/1906 was merged just before the docs check job in Jenkins meaning that it succeeded :smile: Great, no restart necessary.

Given that the only thing I did since the previous written approvals was install a new version of the library, I'll only wait for the system tests to pass before merging the PR.

Thanks for reviews!

**Comment by Gelio** [comment]
System tests passed
https://jenkins.cloudify.co/blue/organizations/jenkins/Stage-UI-System-Test/detail/Stage-UI-System-Test/997/pipeline/

I'm going to merge the PR :smile:

**Comment by qooban** [comment]
> Wait, no, I'm not. There is a
> 
> > Size & check-types
> 
> check that is pending... I'll check it out tomorrow

Hmm... Strange. `Size & check-types` stage will be introduced by https://github.com/cloudify-cosmo/cloudify-stage/pull/1524 , but it is not merged yet to `master`. Maybe @yuvlar already updated GitHub settings to wait for this stage to built successfully?

**Comment by Gelio** [comment]
Yeah, maybe... So far I have rerun the multibranch CI, let's see if that helps :thinking:

**Comment by Gelio** [comment]
Unfortunately, restarting did not help. I'll ask on the ui-ci channel

**Comment by Gelio** [comment]
@yuvlar fixed the problem, I can merge the PR now, which I will do momentarily :slightly_smiling_face:

**Pull request merged**